### PR TITLE
Adds Chem Sprayer to uplink for the Chemist and the Clown

### PR DESCRIPTION
**Pull request by ghost**
lubepocolypse has come

:cl: More Robust Than You
add: Adds Chem Sprayer to uplink for the Chemist and the Clown. Bonus lube bottle included!
/:cl:

### CODE REVIEW

**Comment by lordpidey** [comment]
Uhh...  The title doesn't match the PR.  Did you mean for Janitor and Clown?

**Comment by MoreRobustThanYou** [comment]
WOOPS

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -1272,6 +1272,15 @@ GLOBAL_LIST_EMPTY(uplink_items) // Global list so we only initialize this once.
 	surplus = 20
 	restricted_roles = list("Janitor")
 
+/datum/uplink_item/role_restricted/sprayer
+	name = "Mass Chemical Sprayer"
+	desc = "A chemical sprayer that allows a wide dispersal of selected chemicals. Does not come with chemicals pre loaded into the sprayer.\
+	Bonus Space Lube bottle included!"
+	item = /obj/item/storage/box/syndie_kit/lubepocolypse
+	cost = 16 //maybe get yourself some noslips and a syndiekey or some shit
```
Comment (GunHog) [inline code/modules/uplink/uplink_item.dm]: Noslips do not work with lube!!!

**Comment by GunHog** [comment]
If it were me, I would give this to the chemist and janitor.

**Review by Cyberboss — APPROVED** [review]

**Comment by juanototo2** [comment]
Chemist, janitor and clown

File: code/modules/uplink/uplink_item.dm (hunk 1 above)
Comment (MoreRobustThanYou) [reply code/modules/uplink/uplink_item.dm]: Who said it has to be lube

**Comment by KorPhaeron** [comment]
We removed these from normal play for a reason

3x9 shotgun down the hall of deathchem or lube is terrible

**Comment by ma44** [comment]
>Adding a spray bottle but now it stores 600 reagents and can spray down three tiles at once
👍 👎

**Comment by MoreRobustThanYou** [comment]
@ma44  @KorPhaeron 

> actually managing to produce 600u of deathchems/lube and not getting lynched by security, being locked down by the AI, and toolboxed to death by angry greytiders as soon as you pull the thing out

**Comment by ma44** [comment]
>Implying security would ever arrest a chemist for actually doing something in the lab unless they are looking for valid arrests
>Also not just validing every single kid that gets through the bolted airlocks the AI put down while comboing meth, synaptizine, nicotine and ephedrine

**Comment by KorPhaeron** [comment]
If you think nobody will ever be able to fill it why are you adding it?

**Comment by GuyonBroadway** [comment]
To give you an idea of what this might be like on hippie we added a chemtank system that let you fill up a thing like a water tank with any chem. As a side effect you could fill a fire extinguisher from it and fill the halls with any chem, including lube.

Even with a public chem lab in main't you hardly see anyone do anything beside occasionally lubing the place or filling it with pepper spray.

And remember the whole fukken crew could do this, not just the occasional traitor chemist/clown. The whole fukken crew of hippie who's to do list simply consists of "fuck up someones day".

**Comment by KorPhaeron** [comment]
My distinct memory of what this was like on /tg/ was acid shotguns instant killing hallways of people every round

**Comment by GuyonBroadway** [comment]
Yeah but now acid doesn't apply all its damage instantly so its less terrible.

**Comment by XDTM** [comment]
mutation toxin!

**Comment by GuyonBroadway** [comment]
See that sounds funny as fuck,

**Comment by XDTM** [comment]
until it's _plasmaman_ toxin

**Comment by GuyonBroadway** [comment]
And thus we find the reason for Nanotrasen putting the _oxygenistoxictohumans_ module on a station.

Juuust in case of spontaneous plasmamanification of the crew.

**Comment by Cyberboss** [comment]
Adding something akin to the syringe gun while it's being removed seems to be rocking the boat too much, dontcha think?

**Pull request closed without merging**